suite/tests/CMakeLists.txt

```diff
@@ -4511,6 +4511,71 @@ else (UNIX)
       PASS_REGULAR_EXPRESSION ".*${expect}[ \n]*$")
   endif (X86)
 
+
+  # Cross-arch injection x64 to x86 test using -inject_x64.
+  if(X64)
+    set(inject_x64_dir ${CMAKE_CURRENT_BINARY_DIR}/inject_x64)
+    set(test_bindir suite/tests/bin)
+    file(MAKE_DIRECTORY ${inject_x64_dir})
+    # Unfortunately, while CMake 3.13+ supports create_symlink on Windows, that
+    # only works if the user has the right privileges, so we can't set up
+    # lib dir symlinks like we do for unix cross-arch build-and-test.
+    # We have to make a copy in order to have parallel dirs with the two bitwidths.
```

Comment:
Instead of duplicating everything verbatim, including these long cmoments, it would be nice to share code and comments to reduce maintenance burden and future staleness when one is changed w/o the other.  If you're not comfortable enough w/ CMake to make a shared function or macro, at least add a `XXX` comment up above about sharing all this duplication.